src/components/proposal_list/index.tsx

```diff
@@ -43,6 +31,7 @@ const ProposalList = (props: ProposalListProps) => {
               <TableRow
                 key={row.time}
                 className={classnames('single-row')}
```

Comment:
You need some consistency in your classname naming. Why is it `single-row` with hyphen but then the following classes would be camel case like `clockImage` and `mainContent`. Stick with one and be consistent about it.

In the future we should follow BEM. ex - no need to fix just a quick heads up for next time

`mainContent__time`
`mainContent__proposer`